views/tx.tmpl

```diff
@@ -32,6 +32,26 @@
                         </td>
                     </tr>
                 {{end}}
+                {{if or (eq .Type "Vote") (eq .Type "Revocation")}}
+                    {{range .Vin}}
+                        {{if .Txid}}
+                            <tr>
+                                <td width="90" class="text-right pr-2 h1rem p03rem0 xs-w91">TICKET SPENT</td>
+                                <td><a href="/explorer/tx/{{.Txid}}">{{.Txid}}</a></td>
+                            </tr>
+                        {{end}}
+                    {{end}}
+                {{end}}
+                {{if eq .Type "Ticket"}}
```

Comment:
@chappjc will this convert to sql on the backend, or is the db returning the whole table?

Reply:
The handler for the transactions route will lookup the spending transactions from sql.